docs/user-guide/creating-application/fluxcd.md

```diff
@@ -0,0 +1,118 @@
+
+4. Edit the `dashboard-cm` ConfigMap by clicking **Edit live manifest**.
+
+      * To enable deployments via FluxCD, check if the below entries are present in the ConfigMap (create one if it doesn't exist) and select **Apply changes**.<br>
+
+           ```yaml
+           FEATURE_FLUX_DEPLOYMENTS_ENABLE: " true"
+           FEATURE_LINK_EXTERNAL_FLUX_ENABLE: "true"
+           ```
+
+      ![Figure 4: Editing Live Manifest](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/creating-application/fluxcd/fluxcd-edit-live-manifest.jpg)
+
+      ![Figure 5: Adding Feature Flags](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/creating-application/fluxcd/fluxcd-add-flags.jpg)
+
+5. Restart the deployment: 
+
+      * For OSS Users:
+          1. Navigate to Devtron Resource Browser.
+
+          2. Select the cluster for which you have enabled the feature flags.
+
+          3. Click the **Terminal** tab.
+          
+          4. Restart the deployment using the following command:  
+
+               ```yaml
+               kubectl rollout restart deployment dashboard -n devtroncd 
+               ```
+               <br>
+
+               ![Figure 6: Restarting Deployment](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/creating-application/fluxcd/fluxcd-restart-deployment.gif)
+
+      * For Enterprise Users:
+           1. Go to Resource Browser → (Select Cluster in which you have enabled the feature flags) → Workloads → Deployment
+
+           2. Click the checkbox next to the following Deployment workloads and restart them using the ⟳ button:
+
+                * devtron
+
+                * dashboard
+
+                ![Figure 7: Restart 'devtron' and 'dashboard' deployment workloads](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/kubernetes-resource-browser/devtron-intelligence/restart-deployments.jpg)
+
+6. Perform a hard refresh of the browser to clear the cache:
+
+      * Mac: Hold down Cmd and Shift and then press R.
+
+      * Windows/Linux: Hold down Ctrl and then press F5.
+
+### Step 2: Install FluxCD Controller
+
+After enabling the feature flags, the next step is to install FluxCD Controller in every cluster (including the default cluster) in which you want to deploy the FluxCD applications. To do so, follow the steps below:
+
+1. Navigate to Devtron Resource Browser.
+
+2. Select the cluster for which you have enabled the feature flags.
+
+3. Click the **Terminal** tab.
+
+4. Run the following command to install the FluxCD Controller:<br> 
+
+ ```bash
+ kubectl apply -f https://github.com/fluxcd/flux2/releases/download/v0.35.0/install.yaml
+ ```
+ <br>
+
+ ![Figure 8: Installing FluxCD Controller](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/creating-application/fluxcd/fluxcd-install-controller.gif)
+
+5. After the command is executed successfully, you can deploy or migrate your applications in that cluster through GitOps (via FluxCD).
```

Comment:
There's some depedency of existing applications, check with kartik and mention that